app/src/main/java/com/seay/game_of_thrones/model/GoTCharacter.java

```diff
@@ -0,0 +1,44 @@
+package com.seay.game_of_thrones.model;
+
+public class GoTCharacter {
+
+    private int id;
+    private String character;
+    private String actor;
+    private String description;
+    private String image;
+    private String caption;
+    private String credit;
+
+    public int getId() {
```

Comment:
For DTOs I keep all the member fields public and access them directly without Getters